### DISCUSSION TITLE
`--sub-scale` doesn't work any more on such kind of subtitles, any ideas?

### DISCUSSION THREAD

**Discussion by hooke007** [mpv-player/mpv#15509] [General]
[test.zip](https://github.com/user-attachments/files/18126772/test.zip)
Not sure which version started to fail.
mpv 0.37.0 works

**Comment by llyyr** [comment]
That still won't work because #14234 fixes a bug where sub-scale caused typesetting to also be affected by it. Your subtitle track uses \move tag so it gets detected as a sign.

https://github.com/mpv-player/mpv/pull/15512 for a potential fix, set that option to no
